src/ActiveLogin.Authentication.BankId.AspNetCore/Serialization/BankIdLoginOptionsSerializer.cs

```diff
@@ -26,6 +26,7 @@ public byte[] Serialize(BankIdLoginOptions model)
                     writer.Write(model.AllowChangingPersonalIdentityNumber);
```

Comment:
At the top of this file, there is a constant that needs to be changed whenever we change the content of our opions.
Make it a 2 :)

`private const int FormatVersion = 2;`

Reply:
:O

Reply:
Hehe, it's the convention Microsoft uses. The first field serialized is always a version. If someone has a browser open during release, and we make a breaking change to the "schema", the version ensure we can fail that request.

Reply:
Very nice! I come from java land and I don't think we have conventions like that there 🤔 

.net core-land is much nicer :D